ff_locking.c

```diff
@@ -264,21 +272,23 @@ void FF_LockFAT( FF_IOManager_t * pxIOManager )
 
     configASSERT( FF_Has_Lock( pxIOManager, FF_FAT_LOCK ) == pdFALSE );
 
-    for( ; ; )
+    for (;;) 
     {
-        /* Called when a task want to make changes to the FAT area.
-         * It waits for the desired bit to come high, and clears the
-         * bit so that other tasks can not take it. */
-        xBits = xEventGroupWaitBits( pxIOManager->xEventGroup,
-                                     FF_FAT_LOCK_EVENT_BITS, /* uxBitsToWaitFor */
-                                     pdTRUE,                 /* xClearOnExit */
-                                     pdFALSE,                /* xWaitForAllBits n.a. */
-                                     FF_TIME_TO_WAIT_FOR_EVENT_TICKS );
-
-        if( ( xBits & FF_FAT_LOCK_EVENT_BITS ) != 0 )
+        xEventGroupWaitBits(pxIOManager->xEventGroup,
```

Comment:
Similar issuer here as well. The return value of xEventGroupWaitBits is not checked.